libs/cart/src/reducers/cart-resolve/cart-resolve.reducer.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  DaffCartActionTypes,
+} from '../../actions/public_api';
+import { initialState } from '../cart-initial-state';
+import { DaffCartReducerState } from '../cart-state.interface';
+import { ActionTypes } from '../action-types.type';
+import { DaffCart } from '../../models/cart';
+
+export function cartResolveReducer<T extends DaffCart = DaffCart>(
+  state = initialState,
+  action: ActionTypes
+): DaffCartReducerState<T> {
+  switch (action.type) {
+    case DaffCartActionTypes.ResolveCartAction:
+      return {
+        ...state,
+        resolved: false
+      };
+
+    case DaffCartActionTypes.ResolveCartSuccessAction:
+      return {
+        ...state,
+        cart: {
+          ...state.cart,
+          ...action.payload
+        },
```

Comment:
I don't think this reducer should need to track the cart. That should be handled by the actions that end up triggering the ResolveCartSuccessAction like DaffCartCreateSuccess.

Reply:
good point, changed